lib/bios/menus.robot

```diff
@@ -412,6 +413,48 @@ Reset To Defaults Tianocore
     Read From Terminal Until    ignore.
     Write Bare Into Terminal    y
 
+    # For platforms using PiKVM there is a need to turn on Serial Redirection otherwise most of tests might break.
+    IF    '${DUT_CONNECTION_METHOD}' == 'pikvm'
+        ${menu}=    Read From Terminal Until    Save screenshot
+        IF    '''Continue''' not in '''${menu}'''
+            ${menu}=    Go To Dasharo Main Screen
+        END
+        ${main_menu}=    Parse Menu Snapshot Into Construction    ${menu}    3    1
+        ${dasharo_menu}=    Enter Dasharo System Features    ${main_menu}
+        ${serial_menu}=    Enter Dasharo Submenu    ${dasharo_menu}    Serial Port Configuration
+        ${serial_state}=    Get Option State    ${serial_menu}    Enable COM0 Serial
+        IF    ${serial_state} != ${TRUE}
+            Set Option State    ${serial_menu}    Enable COM0 Serial    ${TRUE}
+        END
+        Press Key N Times    3    ${ESC}
+    END
+
+Go To Dasharo Main Screen
+    [Documentation]    This keyword will press ESC until it will detect the main Dasharo main screen.
+    FOR    ${i}    IN RANGE    0    10
+        ${menu}=    Read From Terminal Until    Save screenshot
+
+        IF    '''Continue''' in '''${menu}'''
+            Return Highlight To Default Position In Setup Menu    ${menu}
+            RETURN    ${menu}
+        ELSE
+            Press Key N Times    1    ${ESC}
+            Sleep    1s
+        END
+    END
+
+Return Highlight To Default Position In Setup Menu
```

Comment:
I checked and it does not work for me, it sets highlight to the `Reset` instead

Reply:
The problem is, that after 1st ESC what you get over telnet when reading until `save screenshot` is the menu, but with no help text visible:

```
QEMU x86 q35/ich9
 QEMU Virtual CPU version 2.5+                       0.00 GHz
 Dasharo (coreboot+UEFI) v0.2.0                      2048 MB RAM @ 200 MHz



   Select Language            <Standard English>

 > User Password Management
 > Device Manager
 > Dasharo System Features
 > One Time Boot
 > Boot Maintenance Manager

   Continue
   Reset





  LCtrl+LAlt+F12=Save screenshot
  ```

Reply:
I suggest that we move it to a separate task, as getting the `Reset to defaults` to work on MSI is going to be tricky...

I suggest we stick with the original automation plan we've had in the configs, and focus on getting these to work, not adding more.

Reply:
https://github.com/Dasharo/open-source-firmware-validation/issues/540